src/DynamoCoreWpf/Controls/DragCanvas.cs

```diff
@@ -241,7 +243,6 @@ protected override void OnMouseLeftButtonDown(MouseButtonEventArgs e)
                 //capture the mouse input even if the mouse is dragged outside the canvas
                 this.CaptureMouse();
                 base.OnMouseLeftButtonDown(e);               
-                e.Handled = true;
```

Comment:
I'm afraid at this place. I think this can cause regressions or bugs. We don't handle event in DragCanvas, because if we click on WorkSpace and InCanvasSearch is open, then WorkSpace should handle click event and close search bar.
I've tried manually and I didn't find any bugs... But it doesn't mean that there is no regressions.
Just wanted to pay your attention here!